editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -160,11 +160,15 @@ export interface CanvasReactReportErrorCallback {
   reportError: (editedFile: string, error: Error, errorInfo?: React.ErrorInfo) => void
 }
 
-export interface CanvasReactErrorCallback extends CanvasReactReportErrorCallback {
+export interface CanvasReactClearErrorsCallback {
   clearErrors: () => void
 }
 
-export type UiJsxCanvasPropsWithErrorCallback = UiJsxCanvasProps & CanvasReactErrorCallback
+export interface CanvasReactErrorCallback
```

Comment:
could you use the same type annotation with the `&` instead of this empty interface extending a list of interfaces?

Reply:
![](https://media3.giphy.com/media/Rlqu7FATdU2mcn1hTt/200.gif)

Reply:
Done